### DISCUSSION TITLE
Select does not accept keyboard input if accessed by tab key

### DISCUSSION THREAD

**Discussion by paul-gaspar** [tusen-ai/naive-ui#3349] [Q&A] [answered]
Hello again,

your select component is a real dream! Best in market, no doubt.

Two little questions:

1) If walk through several inputs by keyboard (tab key), and then set the focus on a select, the control gets a focus ring, but keyboard input is not possible. I have to press the enter key or klick on the control to enable the input field.

This is quite counter intuitive for the end user. Is there a way to fix this? Maybe use the on-focus event for that?

2) It is difficult to find the theme-override key for the outer part of the select component. I need to change the hover-border-color for example. I tried different keys like `Selection`, `BaseSelection` and so on, but nothing works.

Thanks again for your great help!

**Comment by 07akioni** [comment]
For question 2 see here
<img width="1452" alt="image" src="https://user-images.githubusercontent.com/18677354/181055559-25a7872b-2e91-49d1-8f6d-b0ee70e55181.png">

**Reply by paul-gaspar** [reply]
Ok, of couse I've read this page earlier, but then forgot about it again - sorry, my fault. Thank you anyway.

**Comment by 07akioni** [comment, marked as answer]
For question 1 I will think of it to determine whether to change the behavior.

**Reply by paul-gaspar** [reply]
Thank you for the fast response! It would be great if you could do this.
